package/harvester-os/files/usr/sbin/harv-install

```diff
@@ -352,27 +359,35 @@ do_data_disk_format()
     mkfs.ext4 -F -L "$DATA_DISK_FSLABEL" "$HARVESTER_DATA_DISK"
 }
 
+patch_root()
+{
+    # Patching some known issues exists in the rootfs (active.img)
+    # https://github.com/rancher-sandbox/elemental/issues/164
+    [ ! -e ${TARGET}/var/run ] && ln -s /run ${TARGET}/var/run
```

Comment:
This patch could be removed once we have the latest `elemental` in hands. See https://github.com/rancher-sandbox/elemental/issues/164